circle.yml

```diff
@@ -40,10 +26,22 @@ workflows:
                 name: Build folder 🏗
                 command: npm run build
 
-      - lint-markdown-links:
+      - cypress/run:
+          name: Lint Markdown links
           filters:
             branches:
               only: main
+          executor: cypress/base-12
+          requires:
+            - Install
+          # notice a trick to avoid re-installing dependencies
+          # in this job - a do-nothing "install-command" parameter
+          install-command: echo 'Nothing to install in this job'
+          # we are not going to use results from this job anywhere else
```

Comment:
What the motivation for using cypress/run instead of normal docker container with node?

Reply:
caching, all OSS dependencies are present, restores the CircleCI workspace correctly. If you want to recreate this, see the expanded config at circle, it is giant https://app.circleci.com/pipelines/github/bahmutov/cypress-react-unit-test/635/config

this config is 330 lines, expanded 1000

Reply:
Normally the above command would have "parallel" flag, in that case it skips the install. Since we are just reusing workspace from the previous install job, I have to hack skipping the install. I should probably just add a flag to our orb `install: false` to do this better